src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -171,8 +179,13 @@ module sonic-vlan {
 
 				leaf port {
```

Comment:
This leaf includes a custom validation (sonic-ext:custom-validation ValidateVlanMember;), which will be added once the support for custom validation is merged via' this PR,
https://github.com/Azure/sonic-buildimage/pull/6917